### PR TITLE
AzMon integration

### PR DESCRIPTION
**Pull request by shrishrirang**
- Added the azure.appservice.jar which contains AzMon related functionality
- Added a logging.properties and hooked it into the startup command in init_container.sh
- Use org.springframework.boot.loader.PropertiesLauncher for launching the app. This helps us specify a custom class path that contains the AzMon jar.

### CODE REVIEW

**Comment by shrishrirang** [comment]
@jvano , @pagariyaalok

File: shared/logging.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+handlers = com.microsoft.azure.appservice.logging.AsyncFileHandler, java.util.logging.ConsoleHandler
+
+# AzMon logging
+com.microsoft.azure.appservice.logging.AsyncFileHandler.level = FINE
+com.microsoft.azure.appservice.logging.AsyncFileHandler.directory = /var/log/diagnosticLogs
+com.microsoft.azure.appservice.logging.AsyncFileHandler.prefix = application.${workerid}.
```
Comment (jvano) [inline shared/logging.properties]: >${workerid} [](start = 77, length = 11)

double check with Alok if this is needed

**Review by jvano — APPROVED** [review]
:shipit:

File: shared/logging.properties (hunk 1 above)
Comment (pagariyaalok) [reply shared/logging.properties]: This should be good. @ shrirang - can you paste example file name. The format should be like for example : application.92ca53ad8db4fbb93d4d3b7d8ab54dcf8ffecb2d731f25b0e91ad575d7534c3f.2019-04-15.log

---
In reply to: [275581026](https://github.com/Azure-App-Service/java/pull/22#discussion_r275581026) [](ancestors = 275581026)

File: shared/logging.properties [hunk 2]
```diff
@@ -0,0 +1,8 @@
+handlers = com.microsoft.azure.appservice.logging.AsyncFileHandler, java.util.logging.ConsoleHandler
+
+# AzMon logging
+com.microsoft.azure.appservice.logging.AsyncFileHandler.level = FINE
+com.microsoft.azure.appservice.logging.AsyncFileHandler.directory = /var/log/diagnosticLogs
```
Comment (pagariyaalok) [inline shared/logging.properties:5]: >/var/log/diagnosticLogs [](start = 68, length = 23)

can you check the permissions on the log files getting created ?

File: shared/logging.properties (hunk 1 above)
Comment (shrishrirang) [reply shared/logging.properties]: Example: application.dev.2019-04-16.log where dev is the COMPUTERNAME env var

File: shared/logging.properties (hunk 2 above)
Comment (shrishrirang) [reply shared/logging.properties:5]: read access to owner, group, others

**Review by pagariyaalok — APPROVED** [review]
:shipit:

**Pull request merged**